clusterloader2/testing/neg/config.yaml

```diff
@@ -118,3 +118,9 @@ steps:
     Method: WaitForControlledPodsRunning
     Params:
       action: gather
+- name: Wait 15 minutes
```

Comment:
nit: rename to "Wait after deletion" or similar

Reply:
Done.